subcommittees/README.md

```diff
@@ -21,6 +21,7 @@ task).
 * [Finance](finance/README.md)
 * [Lesson Organization and Develpment](lessons/README.md)
 * [Partner Relations and Fundraising](partner-relations/README.md)
+* [Code of Conduct and Policy](policy-CoC/README.md)
```

Comment:
I used both CoC and policy because this is what the committee itself applied in the formation of the policy. The email is also references policy. Again, not sure why, but I'd prefer to leave it to disambiguate, although I agree this muddies the water of what "policy" means.